### PR TITLE
Rk/remove ppa python3.6

### PR DESCRIPTION
**Pull request by rohit-dimagi**
##### SUMMARY

* remove Blocked PPA for python3.6

##### ENVIRONMENTS AFFECTED

* ALL
##### ISSUE TYPE

- Change Pull Request

### CODE REVIEW

**Comment by rohit-dimagi** [comment]
We don't have any Ubuntu 16.04 machines and 18.04 already come with Python 3.6 as default.

File: src/commcare_cloud/ansible/roles/common_installs/tasks/main.yml [hunk 1]
```diff
@@ -23,13 +23,6 @@
   tags:
     - py3
 
-- name: Add Python PPA for bionic and xenial
-  apt_repository: repo='ppa:jonathonf/python-3.6' state=present
```
Comment (snopoke) [inline src/commcare_cloud/ansible/roles/common_installs/tasks/main.yml]: I think we need to proactively remove this from all machines too. If we just remove this what version of python will get installed on new machines? 

The pango one (ppa:jonathonf/gtk3.18) also needs to be replaced.

Comment (rohit-dimagi) [reply src/commcare_cloud/ansible/roles/common_installs/tasks/main.yml]:
```
# python3 --version
Python 3.6.9
# python --version
Python 2.7.17
```

Comment (snopoke) [reply src/commcare_cloud/ansible/roles/common_installs/tasks/main.yml]: OK, that's fine.

File: src/commcare_cloud/ansible/roles/common_installs/tasks/main.yml [hunk 2]
```diff
@@ -11,25 +11,13 @@
   apt_repository: repo='deb http://apt.postgresql.org/pub/repos/apt/ {{ ansible_lsb.codename }}-pgdg main' state=present
   register: add_postgres_repo
 
-- name: Add pango repo for trusty
```
Comment (snopoke) [inline src/commcare_cloud/ansible/roles/common_installs/tasks/main.yml]: Does `libpango1.0-0` still get installed without this? And what version?

Comment (rohit-dimagi) [reply src/commcare_cloud/ansible/roles/common_installs/tasks/main.yml]:
```
# dpkg -l| grep libpango1
ii  libpango1.0-0:amd64                   1.40.14-1ubuntu0.1                amd64        Layout and rendering of internationalized text (transitional package)
# lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.2 LTS
Release:	18.04
Codename:	bionic
```

**Pull request merged**